config_verifier/src/index.ts

```diff
@@ -0,0 +1,520 @@
+    for (var i = 0; i < transceivers.length; i++){
+        var transceiverAddress = transceivers[i];
+
+        // Check to see if it's a wormhole transeiver or not
+        const transceiver = await WormholeTransceiver__factory.connect(transceiverAddress, provider);
+
+        // If it's a wormchain, then setup all of the necessary wormhole specific checks
+        var isWormhole = false;
+        try {
+
+            // If this succeeds, then it's a wormhole transeiver
+            await transceiver.getWormholePeer(1);
+            isWormhole = true;
+
+            var consistencyLevel = await transceiver.consistencyLevel();
+            var nttManager = await transceiver.nttManager();
+
+            // Only going to operate on a transceiver that we know. 
+            transceiverList.push({address: transceiverAddress, isWormhole: isWormhole, transceiver: transceiver, consistencyLevel: consistencyLevel, nttManager: nttManager});
+
+            break;
+        }
+        catch{} // defaults to not adding it
+    }
+
+    managerData['transceivers'] = transceiverList;
+    managerData['enabledTransceiverCount'] = transceivers.length;
+    return managerData;
+}
+
+/*
+Check the following: 
+- Each chain ID corresponds to the expected peer for Wormhole
+- Each decimal corresponds to the expected peer for Wormhole
+- Each chain ID has the proper manager to the expected peer
+*/
+async function checkWormholeTranseivers(chainData){
+ 
+    console.log("Checking transceiver configurations....");
+    for(var target_id in chainData){
+        for (var src_id in chainData){
+
+            var target = chainData[target_id];
+            var src = chainData[src_id];
+            if(target['transceivers'].length == 0){
+                console.log(`ERROR: No found transceivers from Wormhole to verify on target for chain ${target_id}`);
+                continue;
+            }
+
+            if(src['transceivers'].length == 0){
+                console.log(`ERROR: No found transceivers from Wormhole to verify on source for chain ${src_id}`);
+            }
+            if(target['chainId'] == src['chainId']){
+                // One time checks for when they match 
+
+                //// Consistency Level check
+                //// Turn off check for testnet...
+                if(target['transceivers'][0]['consistencyLevel'] == 200 || target['transceivers'][0]['consistencyLevel'] == 201){
+                     console.log(`WARNING: Non-final consistency level for transceiver: ${target['transceivers'][0]['address']}-${target_id}`);
+                }
+
+                // Check the transeiver ownership. Important for sending and receiving messages
+                if(target['transceivers'][0]['nttManager'] != target['managerAddress']){
+                    console.log(`ERROR: Transceiver inconsistent NTTManager Ownership: ${target['transceivers'][0]['address']}-${target_id}`); 
+                }
+
+                continue;
```

Comment:
Could you add a note here as to why the 'chain specific code' below is being skipped in this case?